tests/torch/install_checks.py

```diff
@@ -32,8 +32,12 @@
 levels = 256
 
 if execution_type == "cpu":
-    from nncf.torch.binarization.extensions import BinarizedFunctionsCPU
-    from nncf.torch.quantization.extensions import QuantizedFunctionsCPU
+    try:
+        from nncf.torch.binarization.extensions import BinarizedFunctionsCPU
+        from nncf.torch.quantization.extensions import QuantizedFunctionsCPU
+    except ImportError:
+        from nncf.binarization.extensions import BinarizedFunctionsCPU
+        from nncf.quantization.extensions import QuantizedFunctionsCPU
```

Comment:
Should only run the `try...except` version for the "pip" version of the install checks run - pass the type of the test via CLI to do that.

Reply:
done